src/server/service/kv.rs

```diff
@@ -222,8 +157,8 @@ impl<T: RaftStoreRouter + 'static> tikvpb_grpc::Tikv for Service<T> {
             .and_then(|res| sink.success(res).map_err(Error::from))
             .map(|_| timer.observe_duration())
             .map_err(move |e| {
-                debug!("{} failed: {:?}", LABEL, e);
-                GRPC_MSG_FAIL_COUNTER.with_label_values(&[LABEL]).inc();
+                debug!("{} failed: {:?}", "kv_scan", e);
```

Comment:
why not  `debug!("kv_scan failed: {:?}", e);` directly?

Reply:
I think both are just fine, they are all compiler time stuff, no effect to the runtime.